src/Service.js

```diff
@@ -1,4 +1,5 @@
-const baseUrl = 'http://localhost:5000/';
+ // const baseUrl = 'http://localhost:5000/';
+ const baseUrl = 'https://damp-citadel-25681.herokuapp.com/'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/CSC510-Group-25/feature-hunt/pullRequest?prid=8358184)
```suggestion
 const baseUrl = "https://damp-citadel-25681.herokuapp.com/"
```